tests/DependencyInjection/SchebTwoFactorExtensionTest.php

```diff
@@ -137,6 +137,60 @@ public function load_falsyEnvVarBasedConfig_setConfigValues(): void
         $this->assertNotHasDefinition('scheb_two_factor.security.totp.provider');
     }
 
+    /**
+     * @test
+     */
+    public function load_offOrFalseStringEnvVarBasedConfig_setConfigValues(): void
+    {
+        $yaml = <<<EOF
+trusted_device:
+    enabled: "%env(ENABLE_2FA_OFF_STR)%"
+backup_codes:
+    enabled: "%env(ENABLE_2FA_OFF_STR)%"
+email:
+    enabled: "%env(ENABLE_2FA_OFF_STR)%"
+google:
+    enabled: "%env(ENABLE_2FA_OFF_STR)%"
+totp:
+    enabled: "%env(ENABLE_2FA_OFF_STR)%"
+EOF;
+        $parser = new Parser();
+        $this->extension->load([$parser->parse($yaml)], $this->container);
+
+        $this->assertHasParameter(false, 'scheb_two_factor.trusted_device.enabled');
+        $this->assertNotHasDefinition('scheb_two_factor.security.email.provider');
+        $this->assertNotHasAlias('scheb_two_factor.backup_code_manager');
+        $this->assertNotHasDefinition('scheb_two_factor.security.google.provider');
+        $this->assertNotHasDefinition('scheb_two_factor.security.totp.provider');
+    }
+
+    /**
+     * @test
+     */
+    public function load_onOrTrueStringEnvVarBasedConfig_setConfigValues(): void
+    {
+        $yaml = <<<EOF
+trusted_device:
+    enabled: "%env(ENABLE_2FA_ON_STR)%"
+backup_codes:
+    enabled: "%env(ENABLE_2FA_ON_STR)%"
+email:
+    enabled: "%env(ENABLE_2FA_OFF_STR)%"
```

Comment:
Shouldn't that be on?

Reply:
I wanted to test a 'non-uniform' set of enable/disable settings, but it doesn't really have to be.

Reply:
Yea, please change that. The falsy case has already been covered by the test above, so we'd miss to cover the truthy case :)

Besides that 👍

Reply:
makes sense and sounds good. I've made the change and pushed the branch again